### PR TITLE
Reduce OpenShift CPU and Memory allocations

### PR DESCRIPTION
**Pull request by DerekRoberts**
This should allow four or five PR deployments at once vs the inconsistent two we had before.  Further quota increases would still be helpful!

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Backend](https://nr-spar-90-backend.apps.silver.devops.gov.bc.ca/)
[Frontend](https://nr-spar-90-frontend.apps.silver.devops.gov.bc.ca/)
[Oracle-API](https://nr-spar-90-oracle-api.apps.silver.devops.gov.bc.ca/)

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge-main.yml)

### CODE REVIEW

File: backend/openshift.deploy.yml [hunk 1]
```diff
@@ -31,13 +31,13 @@ parameters:
   - name: FORESTCLIENTAPI_ADDRESS
     value: "https://nr-forest-client-api-prod.api.gov.bc.ca/api"
   - name: CPU_REQUEST
+    value: "25m"
```
Comment (RMCampos) [inline backend/openshift.deploy.yml:34]: `25m` works, but it get really slow. But works.

File: oracle-api/openshift.deploy.yml [hunk 2]
```diff
@@ -19,8 +19,12 @@ parameters:
     value: apps.silver.devops.gov.bc.ca
   - name: CPU_REQUEST
     value: 100m
+  - name: CPU_LIMIT
+    value: 175m
```
Comment (RMCampos) [inline oracle-api/openshift.deploy.yml:23]: Works fine 👍🏻

File: oracle-api/openshift.deploy.yml [hunk 3]
```diff
@@ -19,8 +19,12 @@ parameters:
...
   - name: MEMORY_REQUEST
-    value: 0.5Gi
+    value: 150Mi
+  - name: MEMORY_LIMIT
+    value: 300Mi
```
Comment (RMCampos) [inline oracle-api/openshift.deploy.yml:27]: Works great 👍🏻

**Review by RMCampos — APPROVED** [review]
I tested all of these values in docker-compose file. They work fine :) Great job 🥇

File: backend/openshift.deploy.yml (hunk 1 above)
Comment (DerekRoberts) [reply backend/openshift.deploy.yml:34]: Okay, let's dial that back up! 40m?

File: database/openshift.deploy.yml [hunk 4]
```diff
@@ -18,6 +18,14 @@ parameters:
...
     description: Where to mount the PVC, subpath (e.g. data/)
     value: /var/lib/postgresql
+  - name: CPU_REQUEST
+    value: 50m
+  - name: CPU_LIMIT
+    value: 115m
+  - name: MEMORY_REQUEST
+    value: 25Mi
+  - name: MEMORY_LIMIT
+    value: 50Mi
```
[the remark this answers is not in the document]
Comment (DerekRoberts) [reply database/openshift.deploy.yml:28]: This has been working on OpenShift. I'll try to figure out the consumption difference...

File: backend/openshift.deploy.yml (hunk 1 above)
Comment (DerekRoberts) [reply backend/openshift.deploy.yml]: Seems fine, so let's merge and watch.  We can allocate more to TEST and PROD too, while leaving DEV minimal.

**Review by craigyu — APPROVED** [review]
512Mi -> 40Mi 🤩

**Pull request merged**